plugins/isolation/index.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
```

Comment:
this this JS file intentionally committed?

Reply:
Was some experimental code, surprised it was showing that commit for you since I made it like 3 months ago.